dashboard/app/models/school.rb

```diff
@@ -420,41 +479,51 @@ def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, update_existing
           end
         end
       end
-    end
 
-    future_tense_dry_run = is_dry_run ? ' to be' : ''
-    summary_message =
-      "School seeding: done processing #{filename}.\n"\
-      "#{new_schools.length} new schools#{future_tense_dry_run} added.\n"\
-      "#{updated_schools} schools#{future_tense_dry_run} updated.\n"\
-      "#{unchanged_schools} schools#{future_tense_dry_run} unchanged (school considered unchanged if only update was adding new columns included in this import).\n"\
-      "#{duplicate_schools.length} duplicate schools#{future_tense_dry_run} skipped.\n"\
-      "State CS offerings#{future_tense_dry_run} deleted: #{state_cs_offerings_deleted_count}, state CS offerings#{future_tense_dry_run} reloaded: #{state_cs_offerings_reloaded_count}\n"
-
-    if updated_schools_attribute_frequency.any?
-      summary_message <<
-        "Among updated schools, these attributes #{is_dry_run ? 'will be' : 'were'} updated:\n"\
-        "#{updated_schools_attribute_frequency.sort_by {|_, v| v}.
-          reverse.
-          map {|attribute, frequency| attribute + ': ' + frequency.to_s}.join("\n")}\n"
-    end
-
-    # More verbose logging in dry run
-    if is_dry_run
-      if new_schools.any?
+      future_tense_dry_run = is_dry_run ? ' to be' : ''
+      summary_message =
+        "School seeding: done processing #{filename}.\n"\
+        "#{new_schools.length} new schools#{future_tense_dry_run} added.\n"\
+        "#{updated_schools} schools#{future_tense_dry_run} updated.\n"\
+        "#{errored_schools.length} schools#{future_tense_dry_run} updated failed.\n"\
```

Comment:
I think this line is the only change here, is that right?

Reply:
yep, just whitespace changes